pom.xml

```diff
@@ -44,11 +44,11 @@
   <properties>
     <revision>0.13</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.89</jenkins.version>
+    <jenkins.version>2.176.1</jenkins.version>
     <java.level>8</java.level>
     <!--  override crappy OSS default -->
     <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
-    <kubernetes.client.version>4.1.3</kubernetes.client.version>
+    <slf4jVersion>1.7.26</slf4jVersion>
```

Comment:
this is a higher version that ships with jenkins and as such allows bugs to be introduced by depending on newer APIs that are available.
https://github.com/jenkinsci/jenkins/blob/jenkins-2.176.1/pom.xml#L94
If something needs this version then there is likely to be an incompatability somewhere at runtime

Reply:
Good catch, I'll back this out. I was trying to sync up with what was in the `kubernetes-plugin` pom and assumed it was correct.

Reply:
Upgraded to `4.6.0-2` which uses the slf4j version provided by jenkins core. See https://github.com/jenkinsci/kubernetes-client-api-plugin/pull/8